steps/bitrise-step-qyrus-mobility-test-runner-android/0.0.1/step.yml

```diff
@@ -0,0 +1,125 @@
+title: qyrus-mobility-test-runner-android
```

Comment:
The title should be a human-friendly string, it can contain spaces and capital letters.